server/api/users.js

```diff
@@ -0,0 +1,20 @@
+const router = require('express').Router();
+
+const { User } = require('../db').models;
+
+router.get('/', async (req, res, next) => {
```

Comment:
We should **never expose user information like this. We are exposing emails/usernames/password without any sort of verification of who is asking for this information. This immediately blocks this code from shipping until resolved.**